srcpkgs/sysbench/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'sysbench'
+pkgname=sysbench
+version=1.0.18
+revision=1
+archs="x86_64"
+build_style=configure
```

Comment:
why not use `gnu_configure``? Having an autogen script means it is using gnu autotools.

Reply:
sorry, it is actually `gnu-configure`